charmcraft.yaml

```
@@ -72,3 +75,13 @@ requires:
   send-remote-write:
     interface: prometheus_remote_write
     description: To forward collected metrics to a prometheus backend.
+  logging-consumer:
```

If we can use a relation name different from gagent, then perhaps:
```suggestion
  send-loki-logs:
```
Wdyt @simskij ?

---

I agree. +1 for signalling attentions